src/Test/TestQueries.cs

```diff
@@ -0,0 +1,323 @@
+using System;
+using System.Collections.Generic;
+using Kusto.Language;
+
+namespace Test
+{
+    class TestQueries
+    {
+        
+        public static void tree(string statement)
+        {
+            var query = KustoCode.Parse(statement);
+            string text = query.Text;
+            Console.WriteLine(text);
+            var root = query.Syntax;
+            GenTree.traverse(root, true);
+        }
+        public static Boolean checkNoKeyword(string input_string)
+        {
+            var query = KustoCode.Parse(input_string);
+            var root = query.Syntax;
+            Boolean noKeywordPresent = true;
+            Kusto.Language.Syntax.SyntaxElement.WalkNodes(root, n => {
+                
+                if (n.Kind.ToString() == "PipeExpression" )
```

Comment:
We define this in constant, and use that here instead. It increases readability too.